android/engine/src/main/java/org/smartregister/fhircore/engine/p2p/dao/P2PSenderTransferDao.kt

```diff
@@ -82,7 +83,9 @@ constructor(
         } else {
           highestRecordId
         }
-      Timber.e("Sending ${it.resource.resourceType} with id ====== ${it.resource.logicalId}")
+      Timber.e(
+        "Sending ${it.resource.resourceType} with id ====== ${it.resource.logicalId} and lastUpdated = ${it.resource.meta?.lastUpdated?.time!!}",
+      )
     }
 
     Timber.e("New highest Last updated at value is $highestRecordId")
```

Comment:
Ditto!

Reply:
Check for any other occurrences on the file

Reply:
The log level has been updated to `info`